zcash_keys/src/address.rs

```diff
@@ -176,7 +176,7 @@ impl UnifiedAddress {
         &self.unknown
     }
 
-    fn to_address(&self, net: NetworkType) -> ZcashAddress {
+    pub fn to_address(&self, net: NetworkType) -> ZcashAddress {
```

Comment:
- Needs documentation now that this is in the public API.
- Addition should be documented in the changelog.